simpeg/electromagnetics/natural_source/survey.py

```diff
@@ -152,8 +152,8 @@ def fromRecArray(cls, recArray, srcType="primary"):
         """
         if srcType == "primary":
             src = PlanewaveXYPrimary
-        elif srcType == "total":
-            src = Planewave_xy_1DhomotD
+        # elif srcType == "total":
+        #     src = Planewave_xy_1DhomotD
```

Comment:
we don't currently have a total field formulation, so I think we can leave this commented out for now